src/metpy/io/gempak.py

```diff
@@ -256,14 +256,12 @@ def _interp_logp_height(sounding, missing=-9999):
         press = sounding['PRES'][i]
         hght = sounding['HGHT'][i]
 
-        if press == missing:
+        if press == missing or pbot == missing:
```

Comment:
```suggestion
        if press == missing:
```
This logic is never actually used in its previous implementation, and the change here breaks the loop altogether. I've got this changed locally but wanted to highlight it here in case totally pulling it out has consequences I hadn't considered. If not, pulling it out pleases flake8 and fixes the breakage.